packages/edit-post/src/plugins/navigation-edit-menu-item.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BlockEditorProvider,
+	BlockTools,
+	__unstableBlockToolbarLastItem,
+	__unstableBlockNameContext,
+} from '@wordpress/block-editor';
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { Fragment } from '@wordpress/element';
+import { ReusableBlocksMenuItems } from '@wordpress/reusable-blocks';
+import { __ } from '@wordpress/i18n';
+
+const NavMenuSidebarToggle = () => {
+ 	// Blocks can be loaded into both post and site editors.
+	// We use this to determine which editor we are in so that
+	// we can determine which inspector controls to open.
+	const {
+		isPostEditor,
```

Comment:
There's no need to test isPostEditor, we're always in post editor here, since this plugin is defined in edit-post

Reply:
In my testing, this was also running in the site editor.